SampleProject/VOKCoreDataManagerTests/ManagedObjectImportExportTests.m

```diff
@@ -674,4 +722,13 @@ - (NSArray *)customMapsArrayWithKeyPaths
              [VOKManagedObjectMap mapWithForeignKeyPath:COOL_RANCH_KEYPATH_KEY coreDataKey:COOL_RANCH_DEFAULT_KEY]];
 }
 
+- (NSArray *)customMapsArrayWithoutMicroseconds
+{
+    return @[[VOKManagedObjectMap mapWithForeignKeyPath:FIRST_NAME_DEFAULT_KEY coreDataKey:FIRST_NAME_DEFAULT_KEY],
+             [VOKManagedObjectMap mapWithForeignKeyPath:LAST_NAME_DEFAULT_KEY coreDataKey:LAST_NAME_DEFAULT_KEY],
+             [VOKManagedObjectMap mapWithForeignKeyPath:BIRTHDAY_DEFAULT_KEY coreDataKey:BIRTHDAY_DEFAULT_KEY dateFormatter:[VOKManagedObjectMap vok_dateFormatterWithoutMicroseconds]],
```

Comment:
I guess technically coding standards say this should be split up?  I kind of think it looks better as-is, though.

Still figured I'd point it out.